scripts/generate-lib.js

```diff
@@ -0,0 +1,34 @@
+/*---------------------------------------------------------
+ * Copyright (C) Microsoft Corporation. All rights reserved.
+ *--------------------------------------------------------*/
+
+const prettier = require('prettier');
+const path = require('path');
+const fs = require('fs');
+
+function writeCodeToFile(code, relativeFilePath) {
+  const fileName = path.join(__dirname, relativeFilePath);
+  fs.writeFileSync(
+    fileName,
+    code,
+    prettier.format(result.join('\n'), {
+      parser: 'typescript',
+      ...require('../package.json').prettier,
+    }),
+    { encoding: 'utf-8' },
+  );
```

Comment:
You're passing the "code" as the second argument and the prettier result as the "options", which won't do anything--surprised it doesn't throw an error

```suggestion
  fs.writeFileSync(
    fileName,
    prettier.format(result.join('\n'), {
      parser: 'typescript',
      ...require('../package.json').prettier,
    }),
    { encoding: 'utf-8' },
  );
```